### ISSUE TITLE
[8.0.0beta2] error in Debian owncloud package

### ISSUE THREAD

**Issue by sndrr** [owncloud/core#13793]
I encountered a minor issue when installing the owncloud 8.0.0-beta2 debian package from the OBS repositories into Debian Wheezy 7.8.

    Setting up owncloud-config-apache (8.0.0~beta2-2) ...
    /var/lib/dpkg/info/owncloud-config-apache.postinst: 53: /var/lib/dpkg/info/owncloud-config-apache.postinst: reload_apache: not found
    dpkg: error processing owncloud-config-apache (--configure):
     subprocess installed post-installation script returned error exit status 127
    Errors were encountered while processing:
     owncloud-config-apache

If you change line 53 in `/var/lib/dpkg/info/owncloud-config-apache.postinst` from `reload_apache restart` into `service apache2 restart` it works without problems. See the *patch* below.

```
53c53
<                       reload_apache restart
---
>                       service apache2 restart
```

**Comment by karlitschek** [comment]
@jnweiger

**Comment by jnweiger** [comment]
fixed in isv:ownCloud:community:8.0/owncloud and isv:ownCloud:community:testing/owncloud

commit
----------------------------------------------------------------------------
r57 | jnweiger | 2015-01-30 11:57:56 | 48c14144480a4739adf65a41093e6fc9 | 8.0.0~beta2 | 

- reload_apache() function added.
- apache config FollowSymlinks enabled, needed for FHS.
----------------------------------------------------------------------------
